docs/es/general/emulator-support-and-issues.md

```diff
@@ -5,684 +5,492 @@ description: Guía completa de compatibilidad y soporte de emuladores para Retro
-- ❓ Núcleo de libretro: **fixGB**
-- ❓ Núcleo de libretro: **SameBoy**
-- ❓ Núcleo de libretro: **TGB Dual**
+- ✅ Núcleo de BizHawk: **Gambatte** (fork de speedrunning de Pokémon)
+- ✅ Núcleo de BizHawk: **GBHawk**
 
-#### Game Boy Color
+### Game Boy Color
 
 - ✅ Emulador independiente: **[RAVBA](https://retroachievements.org/download.php#ravba)**
 - ✅ Núcleo de libretro: **Gambatte**
-  - Más recomendado
+  - El más recomendado.
 - ✅ Núcleo de libretro: **Gearboy**
 - ✅ Núcleo de libretro: **mGBA**
 - ✅ Núcleo de libretro: **VBA-M**
-- ✅ Núcleo de BizHawk: **Gambatte** (fork de Pokémon speedrunning)
+- ✅ Núcleo de BizHawk: **Gambatte** (fork de speedrunning de Pokémon)
 - ✅ Núcleo de BizHawk: **GBHawk**
-- ❌ Emulador independiente: **Pizza Boy GBC**
-- No está siendo desarrollado. Problemas de compatibilidad pueden causar problemas con los desbloqueos.
-- ❓ Núcleo de libretro: **DoubleCherryGB**
-- ❓ Núcleo de libretro: **Emux GB**
-- ❓ Núcleo de libretro: **fixGB**
-- ❓ Núcleo de libretro: **SameBoy**
-- ❓ Núcleo de libretro: **TGB Dual**
 
-#### Game Boy Advance
+### Game Boy Advance
 
 - ✅ Emulador independiente: **[RAVBA](https://retroachievements.org/download.php#ravba)**
 - ✅ Núcleo de libretro: **VBA-M**
 - ✅ Núcleo de libretro: **Beetle GBA**
 - ✅ Núcleo de libretro: **VBA Next**
 - ✅ Núcleo de libretro: **mGBA**
-  - Más recomendado
+  - El más recomendado.
 - ✅ Núcleo de BizHawk: **mGBA**
-- ❌ Emulador independiente: **Pizza Boy GBA**
-- No está siendo desarrollado. Problemas de compatibilidad pueden causar problemas con los desbloqueos.
-- ❓ Núcleo de libretro: **gpSP**
-- ❓ Núcleo de libretro: **Meteor**
-- ❓ Núcleo de libretro: **TempGBA**
 
-#### GameCube
+### GameCube
 
 - ✅ Emulador independiente: **[Dolphin](https://retroachievements.org/download.php#dolphin)**
-- ⌛ Núcleo de libretro: **Dolphin**
-  - En pruebas
+  - Se debe usar la versión 2407-68 o mayor.
+  - Verifique que "Activar Renderizado de Doble Núcleo (aumento de velocidad)" esté desactivado ya que no está soportado.
 
-#### Game Gear
+### Game Gear
 
 - ✅ Emulador independiente: **[RAMeka](https://retroachievements.org/download.php#rameka)**
 - ✅ Núcleo de libretro: **Genesis Plus GX**
-- ✅ Núcleo de libretro: **GearSystem**
+- ✅ Núcleo de libretro: **Gearsystem**
 
-#### Genesis / Mega Drive
+### Genesis/Mega Drive
 
 - ✅ Núcleo de libretro: **Genesis Plus GX**
-  - Más recomendado
-- ✅ Núcleo de libretro: **Picodrive**
-- ❌ Núcleo de libretro: **Blastem**
-  - Aunque algunos logros pueden funcionar, el núcleo tiene problemas con la parte de la memoria RAM del juego.
+  - El más recomendado.
+- ✅ Núcleo de libretro: **PicoDrive**
 - ✅ Núcleo de BizHawk: **Genplus-gx** (Genesis Plus GX)
 
-#### Intellivision
+### Intellivision
 
-- ✅ Núcleo de libretro: **FreeIntV**
-  - Se bloquea cuando se reinicia el juego
-  - Algunos bloqueos que podrían estar relacionados con Intellivoice
+- ✅ Núcleo de libretro: **FreeIntv**
+  - Crashea cuando se reinicia el juego.
```

Comment:
I think this change is not good, we understand "crashear" but I think "bloquear" is better for this context